docs/examples/pp_power_flow.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
```

Comment:
Shall we remove this file and only keep the notebook? I like example scripts for development and debugging, but for production (main branch) we chose notebooks.

Reply:
I agree, make sure the same information is also provied in the notebook